### PR TITLE
Add deployer support

### PR DESCRIPTION
**Pull request by taliptako**
Tested working

### CODE REVIEW

File: workspace/Dockerfile [hunk 1]
```diff
@@ -304,9 +308,20 @@ ENV INSTALL_LARAVEL_ENVOY ${INSTALL_LARAVEL_ENVOY}
...
-    && composer global require "laravel/envoy=~1.0" \
+    composer global require "laravel/envoy=~1.0" \
+;fi
+
+#####################################
+# Deployer:
+#####################################
+USER laradock
+
+ARG INSTALL_DEPLOYER=true
```
Comment (appleboy) [inline workspace/Dockerfile]: Set default as `false`

File: README.md [hunk 2]
```diff
@@ -1637,15 +1637,40 @@ It should be like this:
 
 
 
+<br>
 
+<a name="Install-Deployer"></a>
```
Comment (appleboy) [inline README.md]: You forgot to update top menu.

**Comment by appleboy** [comment]
Thanks.

**Comment by jasperf** [comment]
How do you run deployer to deploy your local Laravel app to production using this setup? Do you need to log into the workspace with Docker Machine or something? Reading https://github.com/deployphp/deployer/issues/1636 it seems deployer cannot be used just like that. But perhaps we are using it the wrong way.

**Pull request merged**